### ISSUE TITLE
Docset for objective-c/java APIs

### ISSUE THREAD

**Issue by xster** [flutter/flutter#32377]
Like https://github.com/flutter/flutter/issues/9955 but for https://api.flutter.dev/javadoc/ and https://api.flutter.dev/objcdoc

**Comment by xster** [comment]
@gspencergoog may have some ideas

**Comment by gspencergoog** [comment]
It should be possible to scrape it using Dashing once it's built, and include it in the docset.  I'm sure someone has already developed the Dashing rules for javadoc and objc somewhere...

**Comment by xster** [comment]
Note: specifically https://github.com/technosophos/dashing

We probably have to create dashing.json files for each platform too to map css to their types

**Comment by gspencergoog** [comment]
Yeah, somebody has to, but it seems unlikely that someone else hasn't done that already.

There's also this for javadocs: https://github.com/Kapeli/javadocset#readme
and Jazzy (which supports Dash docsets) for objc: https://github.com/realm/jazzy#readme

The only thing then, is can we merge docsets, which I think is possible, but I'm not sure someone has written the code for it.  Of course, we can just host three separate docsets that people can subscribe to.

**Comment by xster** [comment]
3 separate docsets seem fine. 

Some notes for future, generating docs actually produce a dev/docs/doc/objcdoc/docsets/Flutter.docset which works already. https://api.flutter.dev/objcdoc/docsets/Flutter.xml is a valid link too. Though the `https://docs.flutter.io/objc/docsets/Flutter.tgz` which it points to isn't. https://api.flutter.dev/objcdoc/docsets/Flutter.docset isn't valid either, but https://api.flutter.dev/objcdoc/docsets/Flutter.tgz is.

**Comment by xster** [comment]
We should also make sure the docset name and keyword isn't just `Flutter` and `flutter:` when looking at the iOS engine code.

**Comment by Sunbreak** [comment]
Docset for Java APIs seems to be stale. [FlutterEngine.getInstance()](https://api.flutter.dev/javadoc/io/flutter/embedding/engine/loader/FlutterLoader.html#getInstance--) is deprecated in favor of `FlutterInjector` but `FlutterInjector` is missing on docset
